packages/api/src/sdk-bundle-api/sdk-to-bundle/session-data.ts

```diff
@@ -94,3 +99,15 @@ export interface EarlyRequest {
   startTime: number;
   duration: number;
 }
+
+export interface WebSocketConnectionData {
+  id: number;
+  url: string;
+  events: WebSocketConnectionEvent[];
+}
+
+export interface WebSocketConnectionEvent {
+  timestamp: number;
+  type: "created" | "opened" | "message sent" | "message received" | "closed" | "error";
```

Comment:
Let's standardize on kebab-case to match the other (newer) typings, unless these match some types in the spec